src/hotspot/os/windows/os_windows.cpp

```diff
@@ -3324,16 +3324,27 @@ static size_t large_page_init_decide_size() {
     WARN("Large page is not supported by the processor.");
     return 0;
   }
-
-#if defined(IA32) || defined(AMD64)
-  if (size > 4*M || LargePageSizeInBytes > 4*M) {
+#if defined(IA32)
+    if (size > 4 * M || LargePageSizeInBytes > 4 * M) {
```

Comment:
Please fix the indentation

Reply:
Thank you @djelinski for reviewing this PR.
fixed the indentation.